specification/schemaregistry/data-plane/Microsoft.EventHub/stable/2022-10/schemaregistry.json

```diff
@@ -0,0 +1,615 @@
+              }
+            },
+            "schema": {
+              "$ref": "#/definitions/Error"
+            }
+          }
+        }
+      }
+    },
+    "/$schemaGroups/{groupName}/schemas/{schemaName}:get-id": {
+      "post": {
+        "summary": "Get ID for existing schema.",
+        "description": "Gets the ID referencing an existing schema within the specified schema group, as matched by schema content comparison.",
+        "x-ms-examples": {
+          "Get schema ID": {
+            "$ref": "./examples/OperationSchema_QueryIdByContent.json"
+          }
+        },
+        "operationId": "Schema_QueryIdByContent",
+        "consumes": [
+          "application/json; serialization=Avro",
+          "application/json",
+          "application/json; serialization=Json",
+          "application/schema+json",
+          "application/octet-stream"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "groupName",
+            "in": "path",
+            "description": "Schema group under which schema is registered.  Group's serialization type should match the serialization type specified in the request.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/SchemaName"
+          },
+          {
+            "name": "schemaContent",
+            "in": "body",
+            "description": "String representation (UTF-8) of the registered schema.",
+            "required": true,
+            "schema": {
+              "type": "string",
+              "format": "binary"
+            }
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "204": {
+            "description": "No Content",
+            "headers": {
+              "Location": {
+                "type": "string",
+                "description": "URL location of schema, identified by schema group, schema name, and version."
+              },
+              "Schema-Id": {
+                "type": "string",
+                "description": "References specific schema in registry namespace."
+              },
+              "Schema-Id-Location": {
+                "type": "string",
+                "description": "URL location of schema, identified by schema ID."
+              },
+              "Schema-Group-Name": {
+                "type": "string",
+                "description": "References schema group."
+              },
+              "Schema-Name": {
+                "type": "string",
+                "description": "References schema name."
+              },
+              "Schema-Version": {
+                "type": "integer",
+                "description": "Version of the returned schema."
+              }
+            }
+          },
+          "415": {
```

Comment:
You should generally not document specific error codes. See https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#:~:text=YOU%20SHOULD%20NOT%20document%20specific%20error%20status%20codes%20in%20your%20OpenAPI

Reply:
I think this needs to be added back to avoid a breaking change.

Reply:
Chatted offline - this is probably an okay change as it would only potentially break swagger transforms that shouldn't be there in the first place.